engine/runtime-compiler/src/test/java/org/enso/compiler/test/mock/WithMockCompilerContext.java

```diff
@@ -0,0 +1,160 @@
+package org.enso.compiler.test.mock;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.List;
+import java.util.Set;
+import org.apache.commons.vfs2.FileObject;
+import org.enso.compiler.Compiler;
+import org.enso.compiler.context.CompilerContext.Module;
+import org.enso.compiler.data.CompilerConfig;
+import org.enso.editions.LibraryName;
+import org.enso.pkg.Package;
+import org.junit.rules.TestRule;
+import org.junit.runner.Description;
+import org.junit.runners.model.Statement;
+import scala.jdk.javaapi.CollectionConverters;
+
+/**
+ * A {@link TestRule} to use with the {@link MockCompilerContext}.
```

Comment:
`MockCompilerContext` is an implementation class, not visible in the API. It makes little sense to reference it from API documentation.